gulpfile.js

```diff
@@ -1141,6 +1145,36 @@ gulp.task("jsdoc", function (done) {
   });
 });
 
+gulp.task("types", function (done) {
+  console.log();
+  console.log("### Generating typescript definitions using " + TYPES_INT);
+
+  if (TYPES_INT === "tsc") {
+    var args =
+      `--target ES2020 --allowJS --declaration --outDir ${TYPES_BUILD_DIR} ` +
+      `--strict --esModuleInterop --forceConsistentCasingInFileNames ` +
+      `--emitDeclarationOnly`;
```

Comment:
This really ought to be an array instead, to improve readability. You can simply `join` the elements below instead.